cli/js/fetch_test.ts

```diff
@@ -96,6 +96,7 @@ testPerm({ net: true }, async function responseClone(): Promise<void> {
   const ab = await response.arrayBuffer();
   const ab1 = await response1.arrayBuffer();
   for (let i = 0; i < ab.byteLength; i++) {
+    // @ts-ignore
```

Comment:
what are we ignoring and why?

Reply:
TypeScript doesn't consider indexing ArrayBuffers by number valid, I guess a better alternative might be to create Uint8Array views over these buffers and compare those.